src/components/language-selector.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { navigate } from "gatsby";
+import { Grid, Link, Menu, NavDropDownButton } from "@trussworks/react-uswds";
+import globe from "../images/logos/globe.png";
+
+function LanguageSelector() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  //Hard coding here for now - will update soon
+  const testMenuItemsOne = [
+    <Link to="/en/landing" key="en">
+      English
+    </Link>,
+    <Link to="/es/landing" key="es">
+      Español
+    </Link>,
+    <Link to="/en/landing" key="en">
+      שידיי
+    </Link>,
+    <Link to="/en/landing" key="en">
+      Polski
+    </Link>,
+    <Link to="/en/landing" key="en">
+      Français
+    </Link>,
+    <Link to="/en/landing" key="en">
+      中文
+    </Link>,
+    <Link to="/en/landing" key="en">
+      繁体中文
+    </Link>,
+    <Link to="/en/landing" key="en">
+      ودرا
+    </Link>,
+    <Link to="/en/landing" key="en">
+      ةيبرعلا
+    </Link>,
+    <Link to="/en/landing" key="en">
+      Русский
+    </Link>,
+    <Link to="/en/landing" key="en">
+      বাংলা
+    </Link>,
+    <Link to="/en/landing" key="en">
+      한국어
+    </Link>,
+    <Link to="/en/landing" key="en">
```

Comment:
Don't need these many languages anymore.

Reply:
I've been thinking about it and I think we might want the option to have more than a few languages, since it will require a whole other component - the dropdown menu. If the user only wants 2 or 3 languages, then they can just have that number of links next to each other in the banner. But if the user wants to add 13 languages, it would be hard for them to do so since the header wouldn't fit that many links.

Reply:
Maybe we should have 2 language selector options for the few and many languages use cases

Reply:
Rapi mentioned focusing on only English and Spanish. Maybe you can reiterate your idea to him?

Reply:
Yes let's discuss during our code review tomorrow

Reply:
Justin would like it if we make it so it automatically renders the language dropdown or language links directly in the header based on how many languages there are. So for now I think it makes sense to work with all of the ones included in the CTO house styles figma